src/main/java/net/tazpvp/tazpvp/utils/holograms/StringArrayDataType.java

```diff
@@ -0,0 +1,62 @@
+package net.tazpvp.tazpvp.utils.holograms;
+
+import org.bukkit.persistence.PersistentDataAdapterContext;
+import org.bukkit.persistence.PersistentDataType;
+import org.jetbrains.annotations.NotNull;
+
+import java.nio.ByteBuffer;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+
+public class StringArrayDataType implements PersistentDataType<byte[], String[]> {
```

Comment:
This can go in a different package, preferably the utility one

Reply:
fixed